packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -25,13 +25,12 @@ export const styles = theme => ({
   /* Styles applied to the tag elements, e.g. the chips. */
   tag: {
     margin: 3,
-    '&:last-child': {
-      marginRight: 5,
-    },
```

Comment:
@oliviertassinari Why removing this one?

Reply:
I believe this style is a leftover from me, trying different approaches to handle the spacing between the tags and the textbox. I believe last child never match a tag.